### ISSUE TITLE
hard to read text in spelling suggestions pop-over

### ISSUE THREAD

**Issue by bmesuere** [silvestreh/atom-material-ui#89]
In the pop-over containing the spelling suggestions, the selected item is hard to read. A lighter (white) font-color is probably better here.
![screenshot](https://cloud.githubusercontent.com/assets/481872/9679711/3f18d6c4-52f1-11e5-8afd-cf4b7fb69e2b.png)

**Comment by silvestreh** [comment]
@bmesuere are you using a package for spelling suggestions? can't seem to find them.

**Comment by bmesuere** [comment]
It's a core package: https://atom.io/packages/spell-check

**Comment by silvestreh** [comment]
Yeah, I found it a few minutes after asking :D … I still can't seem to find that popover though. The keyboard shortcut is listed as Cmd+: yet nothing happens on my screen. I do get the underline on misspelled words.

**Comment by am17torres** [comment]
Figured I'd chime in on this one. On OS X if you right click a misspelled word and select correct spelling you'll get the pop up (I'm using spell-check 0.59.0). I also noticed the keybinding says `cmd-:` but the readme says `cmd-shift-:` which when I press works.

<img width="304" alt="screen shot 2015-11-02 at 2 50 33 pm" src="https://cloud.githubusercontent.com/assets/70786/10892433/5f59efe6-8171-11e5-9bcf-bce166841b0b.png">

The contrast for me seems okay using atom material syntax and ui themes. 
<img width="319" alt="screen shot 2015-11-02 at 2 45 54 pm" src="https://cloud.githubusercontent.com/assets/70786/10892277/76dd52bc-8170-11e5-9c13-0098f0557b42.png">

To try and repo the OPs problem, I switched to Syntax Theme Atom Light and switched my accent color to teal, it's a little harder to read but I don't find it as bad.

<img width="305" alt="screen shot 2015-11-02 at 2 49 14 pm" src="https://cloud.githubusercontent.com/assets/70786/10892352/ea8bab96-8170-11e5-842e-80b012db513c.png">
